tests/test_fetch.py

```diff
@@ -24,6 +24,7 @@ def test_fetch_npe2_manifest():
     assert mf.npe1_shim is False
 
 
+@pytest.mark.skip("package looks deleted from pypi")
 def test_fetch_npe1_manifest_with_writer():
     mf = fetch_manifest("example-plugin")
```

Comment:
@tlambert03 You have created these tests. Did you know who is the author of the `example-plugin` and why it was removed from pypi?

Reply:
Ugh sorry @Czaki! Yeah I removed that plugin because I was playing with other things - sorry, didn't realize tests were using it. I can pop it back up,but if we think this test (and others like it) might be critical, we can think about maintaining a dummy plugin within the napari org.

Reply:
This is the package used for the npe1 test. So I do not think that it will require maintenance. 

Byt we could migrate it if you wish.

Reply:
I think we should definitely migrate it to npe2 and keep it up to date in the napari org!

Reply:
But this is test to check if npe1 shims works.

Reply:
We can have two branches then :P

Reply:
After check we could just create it as npe1, upgrade to npe2 and in tests just use older version

Reply:
Happy to keep such a plugin in the napari org, but just want to make sure we document it and what it's for! @Czaki do you want me to put up the plugin and release its two versions?

Reply:
You mean npe1 and npe2 version? Yes.

Reply:
But of course, we could think about a better name if such a need exists.

Reply:
Yeah agreed, I think it should be `test-dummy-plugin` or something. And we should hide it from hub. Or even not add the framework classifier?

Reply:
I think that not adding a classifier could be a good idea.

Reply:
I think we have different ideas here; I was imagining an actual `example-plugin` called such, because I can imagine people looking up "napari example plugin" and it would be an easy way in that's not as "scary" as the cookiecutter.

Maybe this just serves a completely different purpose from the testing, in that case maybe this is not the right conversation :P

Reply:
ooops sorry @brisvag I missed this comment. I think maybe these are two separate things? I always viewed the `cookiecutter` as the default plugin template, but I could imagine maintaining some other example plugin - I'm not sure what you think the benefits of it would be, over the cookiecutter though? 

In general, I think that should be separate to a plugin that we use for testing purposes though since they're different targets. But yeah, happy to consider a richer demo-like plugin!